web/gui/dashboard/src/domains/chart/utils/seconds4human.ts

```diff
@@ -0,0 +1,80 @@
+export const seconds4human = (
+  totalSeconds: number | string, overrideOptions?: {[key: string]: string},
+) => {
+  const defaultOptions: {[key: string]: string} = {
+    now: "now",
+    space: " ",
+    negative_suffix: "ago",
+    day: "day",
+    days: "days",
+    hour: "hour",
+    hours: "hours",
+    minute: "min",
+    minutes: "mins",
+    second: "sec",
+    seconds: "secs",
+    and: "and",
+  }
+
+  const options = typeof overrideOptions === "object"
+    ? { ...defaultOptions, ...overrideOptions }
+    : defaultOptions
+
+  let seconds = typeof totalSeconds === "string"
+    ? parseInt(totalSeconds, 10)
+    : totalSeconds
+
+  if (seconds === 0) {
+    return options.now
+  }
+
+  let suffix = ""
+  if (seconds < 0) {
+    seconds = -seconds
+    if (options.negative_suffix !== "") {
+      suffix = options.space + options.negative_suffix
+    }
+  }
+
+  const days = Math.floor(seconds / 86400)
+  seconds -= (days * 86400)
+
+  const hours = Math.floor(seconds / 3600)
+  seconds -= (hours * 3600)
+
+  const minutes = Math.floor(seconds / 60)
+  seconds -= (minutes * 60)
+
+  const strings = []
+
+  if (days > 1) {
+    strings.push(days.toString() + options.space + options.days)
+  } else if (days === 1) {
+    strings.push(days.toString() + options.space + options.day)
+  }
+
```

Comment:
The same doubt that I wrote in other file can be applied here.

Reply:
same as in the comment above. btw. FYI this file is the migrated version and `web/gui/dashboard/src/dashboard.js` is a file which will be removed completely after the refractor, it's just there so the app can run with both old and new code